interface-definitions/system-ip.xml.in

```diff
@@ -48,6 +48,64 @@
               </leafNode>
             </children>
           </node>
+          <node name="tcp">
+            <properties>
+              <help>IPv4 TCP parameters</help>
+            </properties>
+            <children>
+              <node name="mss">
+                <properties>
+                  <help>IPv4 TCP MSS probing options</help>
+                </properties>
+                <children>
+                  <leafNode name="probing">
+                    <properties>
+                      <help>Attempt to lower the MSS if TCP connections fail to establish</help>
+                      <completionHelp>
+                        <list>enable force</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>enable</format>
```

Comment:
Maybe `enable` is too much here
i.e if we have
```
set system ip tcp mss probing
```
It means that the feature is already `enabled`
But I saw the sysctl implementation and don't know the best solution of how to handle it without `enable`

Reply:
What about 

set system ip mss probing

to enable it and

set system ip mss probing force

to force it?